projects/igniteui-angular/src/lib/grids/filtering/grid-filtering.service.ts

```diff
@@ -37,14 +37,15 @@ export class IgxFilteringService implements OnDestroy {
     private filterPipe = new IgxGridFilterConditionPipe();
     private titlecasePipe = new TitleCasePipe();
     private datePipe = new DatePipe(window.navigator.language);
+    private columnStartIndex = -1;
 
     public gridId: string;
     public isFilterRowVisible = false;
     public filteredColumn = null;
     public selectedExpression: IFilteringExpression = null;
-    public columnToChipToFocus = new Map<string, boolean>();
+    public columnToFocus = null;
+    public focusNext = false;
```

Comment:
Perhaps more meaningful name would be something like `shouldFocusNext` or `shouldFocusRight`.